utils/src/main/kotlin/storage/FileArchiver.kt

```diff
@@ -87,14 +87,14 @@ class FileArchiver(
         zipFile.deleteOnExit()
 
         val zipDuration = measureTime {
-            directory.packZip(zipFile, overwrite = true) { path ->
-                val relativePath = path.toFile().relativeTo(directory).invariantSeparatorsPath
+            directory.packZip(zipFile, overwrite = true) { file ->
+                val relativeFilePath = file.relativeTo(directory).invariantSeparatorsPath
```

Comment:
Because we can now filter by files and directories separately, and in that context I found it useful to clarify which of the two a "path" actually points to. `invariantSeparatorsPath` operates on both files and directories, so that's a flawed comparison IMO.

Reply:
But as I don't feel strongly about it, I've reverted that reaming.